pkg/kubelet/dockershim/helpers.go

```diff
@@ -35,7 +35,8 @@ import (
 )
 
 const (
-	annotationPrefix = "annotation."
+	annotationPrefix            = "annotation."
+	noSuchContainerErrorMessage = "No such cotainer: "
```

Comment:
I am trying to do something similar here. This error message depends on the response from docker daemon. Basically best effort until it breaks. I think it is okay for it to break. But it will break silently (only comes with more error logs).